policyengine_us/variables/household/demographic/person/occupation.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class Occupation(Enum):
+    OTHER = 0
```

Comment:
Other enums make the values text, e.g. `OTHER = "other"` or `MANAGEMENT_BUSINESS_FINANCIAL = "management, business and financial services"` so they display well in the computation tree. This doesn't affect the data package as we'll still be passing in `[OTHER, OTHER, ...]`. Could you change this?